apogee/tools/__init__.py

```diff
@@ -174,3 +188,144 @@ def toApStarGrid(spec):
         return out[0]
     else:
         return out
+
+totalpix = 8575 # total pixels on the apstar grid
```

Comment:
I moved apStarWavegrid to tools/__init__.py from spec/plot.py and imported it back into the latter, then used apStarWavegrid rather than my own code to generate the wavegrid.